### ISSUE TITLE
Print all error messages ?

### ISSUE THREAD

**Issue by rebcabin** [lcompilers/lpython#2468]
I'd like a mode or command-line switch that tells LPython to print all error messages it can find in a file as opposed to the current behavior, which seems to be to stop compiling on the first error.

**Comment by certik** [comment]
Yes, that requires an error resilient parser and semantics. For LFortran this is planned from September 2024, it's not a small project. For LPython we can later implement a similar approach.
